ryven-editor/ryven/main/config.py

```diff
@@ -37,6 +38,7 @@ class Config:
     window_geometry: Optional[str] = None
     window_title: str = 'Ryven'
     qt_api: str = 'pyside2'
+    os.environ['QT_API'] = self.qt_api
```

Comment:
What is your intention here? This code doesn't parse, there is no "self" here.

Reply:
should just be `os.environ['QT_API'] = qt_api`
but hardcoding the flow themes removes the necessity to set the environment variable in the config so it is no longer needed

Reply:
I don't think we need to hard code anything, see what I did on `pr166_tmp`